client/src/components/GoogleAuth.js

```diff
@@ -8,7 +8,7 @@ class GoogleAuth extends React.Component {
       window.gapi.client
         .init({
           clientId:
-            "1098253868729-hbrqmkept759lu1gdsukhtk1jujcl1tr.apps.googleusercontent.com",
+            "434963010544-itr9jvms5cn5i6e4097458nrv17uv5me.apps.googleusercontent.com",
```

Comment:
Not sure why this is showing up as a diff. Have you merged master into your branch recently?

Reply:
I think they haven't pulled in a while? But yeah I merged in the changes for the cliend id to master